features/fields_extensie.feature

```diff
@@ -1,126 +1,570 @@
-    | naam                | waarde        |
-    | plaats.code         | 0518          |
-    | plaats.omschrijving | 's-Gravenhage |
-
-  Scenario: persoon heeft opschorting bijhouding en andere gegevens wordt gevraagd met de fields parameter
-    Als personen wordt gezocht met de volgende parameters
-    | naam                | waarde                          |
-    | type                | RaadpleegMetBurgerservicenummer |
-    | burgerservicenummer | 999992077                       |
-    | fields              | geboorte.plaats                 |
-    Dan bevat de persoon met burgerservicenummer '999992077' de volgende geboorte gegevens
-    | naam                | waarde        |
-    | plaats.code         | 0518          |
-    | plaats.omschrijving | 's-Gravenhage |
-    En bevat de persoon met burgerservicenummer '999992077' de volgende opschortingBijhouding gegevens
-    | naam              | waarde     |
-    | reden.code        | overlijden |
-    | reden.datum.datum | 2015-10-01 |
-    | reden.datum.type  | Datum      |
-    En bevat de persoon met burgerservicenummet '999992077' geen overlijden gegevens
+  - opschortingBijhouding reden wordt altijd meegegeven wanneer deze een waarde heeft.
+
+  Rule: De Fields parameter is verplicht.
+    Scenario: Fields parameter ontbreekt
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999994086 |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999994086                       |
+      Dan heeft de response de volgende gegevens
+      | naam     | waarde                                                                                                      |
+      | type     | https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?#System_Net_HttpStatusCode_BadRequest |
+      | title    | Minimale combinatie van parameters moet worden opgegeven.                                                   |
+      | status   | 400                                                                                                         |
+      | detail   | De parameter fields mag niet leeg zijn.                                                                     |
+      | code     | paramsCombination                                                                                           |
+      | instance | /personen                                                                                                   |
+      En heeft de response een invalidParams met de volgende gegevens
+      | code     | name          | reason                  |
+      | required | fields        | Parameter is verplicht. |
+
+  Rule: De Fields parameter mag alleen valide en unieke attribuutnamen bevatten.
+    Scenario: Fields parameter is leeg
```

Comment:
ik denk dat dit scenario eigenlijk hoort bij de vorige regel (fields is verplicht), hoewel technisch dit een pattern fout is (dus in dat opzicht geen van beide rules illustreert). De fields parameter moet dus niet alleen worden opgenomen, maar ook een waarde hebben: minimaal 1 attribuutnaam bevatten